honeybee_energy_standards/lib/constructions.py

```diff
@@ -0,0 +1,63 @@
+"""Extend the honeybee_energy construction library."""
+from honeybee_energy.construction.opaque import OpaqueConstruction
+from honeybee_energy.construction.window import WindowConstruction
+from honeybee_energy.lib.constructions import _idf_opaque_constructions, _idf_window_constructions
+
+import os
+import json
+
+
+# load the standards gem data of construction to Python dictionaries.
+_data_dir = os.path.join(os.path.dirname(__file__), '../data')
+
+_opaque_dir = os.path.join(_data_dir, 'opaque_construction.json')
```

Comment:
You have to include all these JSON files in MANIFEST.in file to ensure they will be part of the package. Also add the key to `setup.py` to include package data.

Reply:
Yes, I did this before. I just forgot to do it this time.